src/Identifier/OrmIdentifier.php

```diff
@@ -23,7 +23,9 @@ class OrmIdentifier extends AbstractIdentifier
 {
 
     use LocatorAwareTrait;
-    use PasswordHasherTrait;
+    use PasswordHasherTrait {
+        getPasswordHasher as private traitGetPasswordHasher;
```

Comment:
Wouldnt protected suffice? This way it could be reused from extension points if needed.